ddtrace/internal/writer/writer.py

```diff
@@ -327,7 +331,7 @@ def write(self, spans=None):
             self.flush_queue()
 
     def _write_with_client(self, client, spans=None):
-        # type: (WriterClientBase, Optional[List[Span]]) -> None
+        # type: (WriterClientBase, Optional[List[Union[Span, LLMObsSpanEvent]]]) -> None
```

Comment:
If you're changing type signatures, I suggest turning them into annotations as opposed to comments.